### ISSUE TITLE
Control rendering of `Field` container element

### ISSUE THREAD

**Issue by Kukkimonsuta** [davidkpiano/react-redux-form#22]
`Field` is trying to be smart and render container element only when there is more than one child element. I'd like it to be even smarter and also render container element for cases where we've already assigned some other properties to it (for instance `className` or `id`).

```jsx
<Field model="filter.email" className="field six wide">
    <input type="text" placeholder="E-mail" />
</Field>
```

Current output (`className` is lost):
```html
<input type="text" placeholder="E-mail" name="filter.email" data-reactid=".0.0.0">
```

Expected output:
```html
<div class="field six wide" data-reactid=".0.0.0">
    <input type="text" placeholder="E-mail" name="filter.email" data-reactid=".0.0.0.$/=11">
</div>
```

**Comment by davidkpiano** [comment]
Interesting suggestion. The current logic is this:

- If the `<Field>` contains only one child, render only that (smartly cloned) child.
- Else if the `<Field>` contains more than one child, render a `<div>` since React can't render multiple children without a parent node (unfortunately).

So this will work as expected: 

```js
<Field model="foo.bar" className="field">
  <label>Test</label>
  <input type="text" />
</Field>

// renders to...
<div class="field">
  <label>Test</label>
  <input type="text" />
</div>
```

So we can update the logic so that if a `className` prop exists on the `<Field>`, generate a `<div>` regardless of the number of children. Is that what you're expecting?

If so, I'll add it (or feel free to PR). A simple workaround is this:

```js
<Field model="...">
  <div className="field six wide">
    <input type="text" placeholder="E-mail" />
  </div>
</Field>
```

**Comment by Kukkimonsuta** [comment]
Yes, that's exactly what I'd like to see. I'm currently using the workaround, but it feels too verbose for no good reason.

There are, however more properties that should be accounted in this feature, ideally anything that would render to the `div` if there were more elements. The list of all `React` supported attributes should be here https://facebook.github.io/react/docs/tags-and-attributes.html#supported-attributes . I'm not sure how to get to that list programatically though.

**Comment by davidkpiano** [comment]
Just a thought: what if a `component={...}` prop was passed in to `Field`? That seems to be the simplest solution for this.

```
// Will not render a 'div' wrapper
<Field model="...">
  <input />
</Field>

// Will render a 'div' wrapper because of 'className'
<Field model="..." className="field six wide">
  <input />
</Field>

// Will also render a 'div' wrapper
<Field model="..." component="div">
  <input />
</Field>

// Will render a View wrapper (useful for React Native)
<Field model="..." component={View}>
  <input />
</Field>
```

**Comment by Kukkimonsuta** [comment]
That's very good idea actually! More control over rendering is always welcome :+1:

**Comment by davidkpiano** [comment]
Alright, published as 0.6.1! https://github.com/davidkpiano/react-redux-form/releases/tag/v0.6.1

**Comment by Kukkimonsuta** [comment]
Works great, thank you :cookie:
